Saw this locally like so after a recent Docker Desktop update:
```
% docker buildx ls
NAME/NODE       DRIVER/ENDPOINT             STATUS  BUILDKIT                              PLATFORMS
default         docker                                                                    
  default       default                     running v0.11.7-0.20230525183624-798ad6b0ce9f linux/arm64, linux/amd64, linux/amd64/v2, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6
desktop-linux * docker                                                                    
  desktop-linux desktop-linux               running v0.11.7-0.20230525183624-798ad6b0ce9f linux/arm64, linux/amd64, linux/amd64/v2, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/mips64le, linux/mips64, linux/arm/v7, linux/arm/v6
% make            
Using Docker Buildx builder "desktop-linux" with build flags "--push --platform=linux/arm64,linux/amd64".
DOCKER_BUILDKIT=1 docker buildx build --push --platform=linux/arm64,linux/amd64 ...
[+] Building 0.0s (0/0)                                                                                                                                                                                                                               
ERROR: multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")
make: *** [...] Error 1
```

--- a/Makefile.docker
+++ b/Makefile.docker
@@ -14,7 +14,7 @@ ifdef ARCH
   # Default to multi-arch builds, always create the builder for all the platforms we support
   DOCKER_PLATFORMS := linux/arm64,linux/amd64
   DOCKER_BUILDER := $(shell docker buildx ls | grep -E -e "[a-zA-Z0-9-]+ \*" | cut -d ' ' -f1)
-  ifeq ($(DOCKER_BUILDER),default)
+  ifneq (,$(filter $(DOCKER_BUILDER),default desktop-linux))
     DOCKER_BUILDKIT_DRIVER :=
     ifdef DOCKER_BUILDKIT_IMAGE
       DOCKER_BUILDKIT_DRIVER := --driver docker-container --driver-opt image=$(DOCKER_BUILDKIT_IMAGE)

